newIDE/app/src/ObjectEditor/Editors/SpriteEditor/index.js

```diff
@@ -246,6 +248,13 @@ class AnimationsListContainer extends React.Component<
     this.forceUpdate();
     this.props.onSizeUpdated();
     if (this.props.onObjectUpdated) this.props.onObjectUpdated();
+
+    // Scroll to the bottom of the list
+    setTimeout(() => {
+      if (this._scrollView.current) {
+        this._scrollView.current.scrollToBottom();
+      }
+    }, 100); // Wait for the list to be updated.
```

Comment:
Use the exact same comment everywhere so this comment is "greppable" in the codebase